src/content/plugins/hashed-module-ids-plugin.md

```diff
@@ -18,6 +19,7 @@ new webpack.HashedModuleIdsPlugin({
 
 This plugin supports the following options:
 
+- `context`: The context directory for creating names.
```

Comment:
Reading webpack source code it seems it has to be an absolute path. Could you please add a note about it?